docs: updates to dashboards descriptions and grafana tasks (#5591)
strimzi/strimzi-kafka-operator: documentation/modules/metrics/ref-prometheus-metrics-config.adoc

// This assembly is included in the following assemblies:
//
// metrics/assembly-metrics-config-files.adoc

[id='ref-metrics-prometheus-metrics-config-{context}']

= Example Prometheus metrics configuration

[role="_abstract"]

> Strimzi uses the link:https://github.com/prometheus/jmx_exporter[Prometheus JMX Exporter^] to expose JMX metrics through an HTTP endpoint, which is then scraped by the Prometheus server.

Reviewer: ```suggestion
Strimzi uses the link:https://github.com/prometheus/jmx_exporter[Prometheus JMX Exporter^] to expose metrics through an HTTP endpoint, which can be scraped by the Prometheus server.
```

// This assembly is included in the following assemblies:
//
// metrics/assembly-metrics-config-files.adoc

[id='ref-metrics-prometheus-metrics-config-{context}']

= Example Prometheus metrics configuration

[role="_abstract"]

> Strimzi uses the link:https://github.com/prometheus/jmx_exporter[Prometheus JMX Exporter^] to expose JMX metrics using an HTTP endpoint, which is then scraped by the Prometheus server.

Reviewer 2: ```suggestion
Strimzi uses the link:https://github.com/prometheus/jmx_exporter[Prometheus JMX Exporter^] to expose JMX metrics through an HTTP endpoint, which is then scraped by the Prometheus server.
```

> Grafana dashboards are dependent on Prometheus JMX Exporter relabeling rules, which are defined for Strimzi components as custom resource configuration.

Reviewer 2: ```suggestion
Grafana dashboards are dependent on Prometheus JMX Exporter relabeling rules, which are defined for Strimzi components in the custom resource configuration.
```
